docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,399 @@ Risk factor mediation
+
+For example, in a hypothetical smoking simulation, if one of our interventions
+could directly remove tar from the lungs after someone had smoked, we would need
+to model lung tar.
+
+.. image:: necessary-mediator.drawio.svg
+
+But if our interventions only affect smoking itself, there is no need to model lung tar;
+it is implicit between smoking and lung cancer.
+
+.. image:: unnecessary-mediator.drawio.svg
+
+There are some potential reasons to model a mediator, even when it is not strictly necessary
+according to these criteria.
+For example, it could be an outcome worth reporting in its own right, or the relevant effect
+sizes could be easier to find
+(risk to mediator and mediator to outcome instead of risk to outcome directly).
+
+If we can ignore one mediator, we can also ignore a cluster of mediators with arbitrary complexity,
+so long as none of them are being acted on by our intervention (except through the modeled risk):
+
+.. image:: unnecessary-mediators.drawio.svg
+
+.. todo::
+  This section describes the factors to consider when deciding whether to model a mediator,
+  or leave it out of the model entirely.
+  It does **not** cover the decision of whether or not to model risk-risk effects or mediation,
+  *given* that all the risks/outcomes involved are included in the model.
+  In the past we have sometimes skipped modeling mediation even between modeled attributes;
+  we have a rough conceptual framework for when this is okay, but have not formalized
+  anything yet.
+
+  A related (though different) question is explored :ref:`on the Risk Correlation page <risk_correlation_when_important>`.
+
+Risk effects in the presence of mediation
+-----------------------------------------
+
+When mediation is modeled **alongside a direct, non-mediated effect**,
+the direct risk effect must be adjusted for the mediator to avoid
+double-counting.
+If there is no direct effect (the entire effect is mediated by a modeled factor),
+nothing special needs to be done.
+
+For example, consider this diagram of **modeled** causes and risks,
+inspired by the cardiovascular disease simulation:
+
+.. image:: bmi-sbp-mediation.drawio.svg
+
+BMI here is body mass index, SBP is systolic blood pressure, and IHD
+is ischemic heart disease.
+We see that BMI has an effect on IHD mediated by SBP, but also has
+a direct effect.
+
+Imagine we found studies describing the effects in this diagram
+by reporting relative risks (RRs).
+To inform the direct BMI→IHD arrow, we'd need the RR of BMI
+on IHD to be adjusted for SBP, in order to isolate the direct effect.
+If we used an RR of BMI on IHD that was not adjusted for SBP, the mediated
+part of the effect would be double-counted in our model: once in the direct
+path and once in the mediated path.
+This logic extends naturally to the situation where
+there are multiple mediated pathways in addition to a direct effect:
+for the direct effect, we need an RR adjusted for **all** mediated pathways.
```

Comment:
I find the concept of a total effect being comprised of a direct/non-mediated and mediated effect can be helpful in describing this concept, where the total effect would be the measured association between BMI->IHD adjusted for confounders; direct would be BMI -> IHD adjusted for confounders and SPB, and the mediated effect would be the magnitude of the pathway through BMI->SBP (adjusted for confounders) and SBP->IHD adjusted for confounders including BMI. 

I think your description is clear and does a great job, but formalizing the terms can be helpful for conversation, etc. if you feel like adding it in!